src/components/Flyout/useContainScroll.ts

```diff
@@ -0,0 +1,38 @@
+import { MutableRefObject, useEffect } from "react";
+
+export const useContainScroll = (overlayRef: MutableRefObject<HTMLDivElement | null>, { isDisabled = false }) => {
+    useEffect(() => {
+        const scrollHandler = (event: WheelEvent) => {
+            const prevent = function () {
+                event.stopPropagation();
+                event.preventDefault();
+                return false;
+            };
+            const overlay = overlayRef.current as HTMLDivElement;
+            const deltaY = event.deltaY;
+            const up = deltaY < 0;
+            const scrollTop = overlay.scrollTop;
```

Comment:
Not sure if this could lead to an error if `overlay` is null
```suggestion
            const scrollTop = overlay?.scrollTop;
```

Reply:
fixed with if condition